audit_report/3.1.0/changes/topics/sphincsplus.yml

```diff
@@ -20,4 +20,10 @@ patches:
 
 # Add SPHINCS+ speed run to the CLI tests  (@Jack Lloyd)
 - commit: 59cb03996b6d308485b847b0aade46e6d124a29c  # https://github.com/randombit/botan/commit/59cb03996b6d308485b847b0aade46e6d124a29c
-  classification: unspecified
+  classification: info
+  auditer: reneme
```

Comment:
I also reviewed this patch in #80. Shouldn't be dramatic, though ;)

Reply:
Mhm, I feared that. Unfortunately for me, you first merged your branch before you approved mine. So, I'll have to resolve that conflict. Karma, I guess. 🤡